ClimbingGearBackend/Startup.cs

```diff
@@ -52,14 +62,20 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         app.UseSwaggerUI(c => c.SwaggerEndpoint("/swagger/v1/swagger.json", "ClimbingGearBackend v1"));
         app.UseMigrationsEndPoint();
       }
+      else
+      {
+        app.UseExceptionHandler("/Error");
+        // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
+        app.UseHsts();
+      }
 
       app.UseHttpsRedirection();
 
       app.UseRouting();
 
       app.UseAuthentication();
-      app.UseIdentityServer();
       app.UseAuthorization();
+      app.UseIdentityServer();
```

Comment:
I remember you mentioning something about auth not working in the test environment. Was this the fix for that?

Reply:
No, auth doesn't working because the OdicController that handles it is a mystery to me. I could setup a temporary User controller that handed out tokens but would have to delete all of it before I deployed this somewhere. I may have to do that just to get the looking up of the current user code working though.